docs/experimental-features/beaconing-detection.asciidoc

```diff
@@ -0,0 +1,50 @@
+[[beaconing-detection]]
+== Beaconing detection
+
+This feature provides an early warning system for command and control (C2) beaconing activity. It monitors network traffic for indicators of compromise (IOCs), and provides analytics to aid your threat hunting.
+
+[discrete]
+=== Deploy the Beaconing detection package
+
+To deploy Beaconing detection in your environment, follow https://github.com/elastic/detection-rules/blob/main/docs/experimental-machine-learning/beaconing.md[these steps].
+
+The release package includes dashboards for monitoring beaconing activity in your environment. Users can review signals via a Lens dashboard called `Beaconing discovery`.
+
+[role="screenshot"]
+image::images/beaconing-detection-1.png[]
+
+[discrete]
+=== Feature details
+
+The feature uses a {ref}/transforms.html[transform] to segment network data by host and process name, then runs scripted metric aggregations on the host-process name pairs. For a given time window, the scripted metric aggregation checks each partition for the following:
+
+* Periodicity of signals, accounting for jitter.
+* Low variation of bytes sent from source to destination.
+* Low variation of bytes sent from destination to source.
+
+The transform also filters out trusted applications and IPs in an attempt to reduce false positives. The transform outputs information about the process and host indicators, and the detection. It runs every hour to poll for new beacons.
+
+[discrete]
+=== Further customizations
+
+Advanced users can also tune the scripted metric aggregation's parameters, such as jitter percentage or time window. To overwrite the default parameters, delete the transform, change the parameters, and restart. The configurable parameters are as follows:
```

Comment:
Unclear what the user needs to "restart" here -- the transform? an instance of Elasticsearch or Kibana?
```suggestion
Advanced users can also tune the scripted metric aggregation's parameters, such as jitter percentage or time window. To overwrite the default parameters: delete the transform, change the parameters, and restart. The configurable parameters are:
```

Reply:
Good question, I'll clarify with the dev.